mutiny-core/src/fees.rs

```diff
@@ -176,6 +176,7 @@ fn num_blocks_from_conf_target(confirmation_target: ConfirmationTarget) -> usize
     match confirmation_target {
         // Background is VERY lax and may never confirm if used directly
         // it is only meant for lower ranges of transaction to enter mempool
+        ConfirmationTarget::MempoolMinimum => 1008,
```

Comment:
nit: the comment above is meant for the `Background`, though I am now curious what this new one is supposed to be used for.

Reply:
Moved the comment and added one for `MempoolMinimum`